SystemManagement/SystemManagement.csproj

```diff
@@ -184,6 +194,10 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\CompanyStudio\CompanyStudio.csproj">

```

Comment:
We shouldn't add a dependency on another project. The reason why it appears this happened is because you used the Loader from Company Studio. We'll actually need to copy/paste it to the System Management project and remove this reference.

Reply:
Fixed